app/controllers/admin/schools/participants_controller.rb

```diff
@@ -7,12 +7,10 @@ class Schools::ParticipantsController < Admin::BaseController
     before_action :set_school
 
     def index
-      @participants = User.where(id: @school.participant_profiles
-                                            .active
-                                            .ecf
-                                            .select(:user_id))
-                          .order(:full_name)
-                          .includes(participant_profile: %i[cohort school])
+      @participants = @school.active_participants.order(:full_name).includes(
+        early_career_teacher_profile: [:mentor, school_cohort: %i[school cohort]],
+        mentor_profile: [school_cohort: %i[school cohort]],
+      )
```

Comment:
This is going to include NPQ ppts, we've lost the `ecf` scope.

Reply:
I doubt it, participants come from `school_cohorts` on the `school`. Which cannot have anything NPQ related.